invenio_resources/controller.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio Resources module to create REST APIs."""
+import uuid
+
+from invenio_db import db
+from invenio_indexer.api import RecordIndexer
+from invenio_pidstore import current_pidstore
+from invenio_records.api import Record
+from invenio_search import RecordsSearch
+
+from .search import es_search_factory
+from .config import lt_es7, obj_or_import_string
+
+# TODO: Is there a generic InvenioResourceController class needed?
+class RecordController:
```

Comment:
There is no controller in `Flask-Resources`. Since the resources might contain the business logic for simple cases, there is no need to add that -complexity- there.